url.bs

```diff
@@ -3484,6 +3486,9 @@ constructor steps are:</p>
  <li><p><a for=URLSearchParams>Initialize</a> <a>this</a> with <var>init</var>.
 </ol>
 
+<p>The <dfn attribute for=URLSearchParams><code>size</code></dfn> getter steps are to return
+<a>this</a>'s <a for=URLSearchParams>list</a>'s <a for=list>size</a>.
```

Comment:
I may have missed the nuance here. My interpretation of `size` is that it should return the total number of elements that would be returned by the iterator. Is that understanding correct? And if not, I think that's what it should be.

/cc @lucacasonato

Reply:
Yeah that is the proposal as per the last set of comments in #163. Note that I haven't yet asked all browser implementers for feedback. I've been meaning to but some other things have higher priority.